### PR TITLE
klient/machine: add tests to machine group ssh handler

### PR DESCRIPTION
**Pull request by ppknap**
This PR contains missing tests for machine SSH handler.

Depends on: ~#10383~

**Code coverage**:  57.6% of statements

## How Has This Been Tested?
Unit tests.

## Screenshots (if appropriate):
none

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: go/src/koding/klient/machine/machinegroup/ssh_test.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+				clienttest.TurnOnAddr(),
+				machine.Addr{
+					Network:   "ip",
+					Value:     fakeHost,
+					UpdatedAt: time.Now(),
+				},
+			},
+		},
+	}
+	if _, err := g.Create(createReq); err != nil {
```
Comment (cihangir) [inline go/src/koding/klient/machine/machinegroup/ssh_test.go]: first to note: i didnt follow the whole flow. Would creating this with fake host twice cause problems? ie db unique index issues etc

File: go/src/koding/klient/machine/machinegroup/ssh_test.go [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+			id: {
+				clienttest.TurnOnAddr(),
+				machine.Addr{
+					Network:   "tunnel",
+					Value:     l.Addr().String(),
+					UpdatedAt: time.Now(),
+				},
+				machine.Addr{
+					Network:   "ip",
+					Value:     "53.23.123.4",
```
Comment (cihangir) [inline go/src/koding/klient/machine/machinegroup/ssh_test.go]: is this a fake IP?

File: go/src/koding/klient/machine/machinegroup/ssh_test.go (hunk 1 above)
Comment (ppknap) [inline go/src/koding/klient/machine/machinegroup/ssh_test.go]: no, machine ID, is a unique index (`id` field L38). When we call `machine list`, the list of machines is sent to the klient which then calls `g.Create` using simillar `CreateRequest`.

- If called twice and machine address doesn't change, nothing will happen.
- If called twice and machine address change, new address will be added to this machine (this handles machines with dynamic IPs)

This logic is here to have a machine into which we will be SSH-ing in.

File: go/src/koding/klient/machine/machinegroup/ssh_test.go (hunk 2 above)
Comment (ppknap) [inline go/src/koding/klient/machine/machinegroup/ssh_test.go]: yes, I made it up :)

**Review by cihangir — APPROVED** [review]

**Review by rjeczalik — APPROVED** [review]

**Pull request merged**